src/prototype_room_creepbuilder.js

```diff
@@ -122,7 +97,7 @@ Room.prototype.spawnCheckForCreate = function(creepsConfig) {
     var creep = this.memory.queue[0];
```

Comment:
i would kie to see some change here: https://github.com/TooAngel/screeps/pull/54/files#diff-72fe2da08c0bdf7fdd904d0252f0cd72R95
```js
  // save priority in queue object
  this.memory.queue = _.sortBy(_.map(this.memory.queue, priorityQueue), 'priority');

```